src/import-data/ImportDataDestination.ts

```diff
@@ -220,14 +219,20 @@ export const ImportDataDestination = (props: ImportDataDestinationProps): ReactN
             }),
           ]),
       ]),
-      importMayTakeTime && div({ style: { marginTop: '0.5rem', lineHeight: '1.5' } }, [importMayTakeTimeMessage]),
-      !!selectedWorkspace &&
-        h(Fragment, [
-          h(WorkspacePolicies, { workspace: selectedWorkspace }),
-          !!isProtectedData &&
-            isAzureWorkspace(selectedWorkspace) &&
-            div([p(['Importing this data may add:']), ul([li(['Additional access controls'])])]),
+      importMayTakeTime &&
+        div({ style: { paddingLeft: '0.5rem', paddingRight: '0.5rem', paddingTop: '0.5rem', lineHeight: '1.5' } }, [
+          importMayTakeTimeMessage,
         ]),
+      !!selectedWorkspace &&
+        h(WorkspacePolicies, {
+          workspace: selectedWorkspace,
+          noCheckboxes: true,
+          policiesLabel: 'Policies on this workspace',
+          endingNotice:
+            isProtectedData && isAzureWorkspace(selectedWorkspace)
+              ? div(['Importing this data may add additional access controls'])
```

Comment:
Is it possible to find out if importing will result in change of access controls? Or maybe it's hard/expensive to do and/or we don't think users will want to know?

Reply:
It is difficult to find out for PFB imports, which can contain data from multiple snapshots. The PFB file is parsed and policies linked by the Workspace Data Service in the destination workspace.

Here, before a workspace is selected, we don't have a WDS available.

Reply:
I do not know… I didn't change the logic of when this displays.